src/styles/styles.css

```diff
@@ -1,5 +1,306 @@
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  padding-left: 0;
+  align-items: center;
+}
+
+.comment-btn,
+.remove-btn,
+.edit-btn {
+  font-family: var(--RobotoSlab);
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+  border: none;
+  cursor: pointer;
+  background-color: transparent;
+}
+
+.book-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book-info {
+  width: 40%;
+}
+
+.book-info p {
+  padding-left: 7px;
+}
+
+.item-progress {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  font-family: var(--Montserrat);
+}
+
+.main-levels {
+  font-family: var(--RobotoSlab);
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.circle {
+  margin-right: 3.46%;
+  border: 1px solid rgb(204, 200, 200);
+  border-radius: 50%;
+  padding: 0.5rem;
+}
+
+.main-task {
+  font-family: var(--RobotoSlab);
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.Update-progress {
+  font-family: var(--RobotoSlab);
+  padding: 0.8em 1.4rem;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+  border-radius: 3px;
+  background-color: var(--azure);
+  border: none;
+}
+
+.Line {
+  width: 90%;
+  border: solid 1px var(--white);
+  margin: 0 auto;
+}
+
+.form-container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.form-title {
+  font-family: var(--Montserrat);
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.complete-task {
+  font-family: var(--Montserrat);
+  margin: 0;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.level-one {
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px var(--white);
+}
+
+.level-two {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px var(--white);
+}
+
+.task-progress {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.completed {
+  margin: 0;
+  border-radius: 4px;
+  border: none;
+  color: var(--very-light-pink);
+}
+
+.levels {
+  margin-right: 4rem;
+}
+
+.hr {
+  border: 1px solid gray;
+  margin: 0 3rem;
+}
+
+.new-book {
+  padding: 0 3rem 0.5rem 3rem;
+  font-family: var(--RobotoSlab);
+  color: #a4a4a4;
```

Comment:
- [ ] Similarly here, [as suggested](https://github.com/mohammadnewton/bookstore/pull/5#discussion_r1306561142), please make sure to use the given text color which is `#888888`